packages/app/scripts/start.js

```diff
@@ -51,7 +54,7 @@ function formatMessage(message) {
 function clearConsole() {
   // This seems to work best on Windows and other systems.
   // The intention is to clear the output so you can focus on most recent build.
-  process.stdout.write('\x1bc');
+  // process.stdout.write('\x1bc');
```

Comment:
@CompuIves Should we completely remove `clearConsole()`? I kind of get its purpose, but on Windows all the previous output is completely lost, which is frustrating when you'd want to look at previous warnings / errors.

Reply:
Hmm, I see. I liked the `clearConsole` but I think it makes sense to not do it in Windows. Maybe `require('os').platform() === 'darwin'` check?

Reply:
Disabled by default on Windows, added `CLEAR` environment variable to override default behavior.